requirements.txt

```diff
@@ -2,6 +2,7 @@ ConfigArgParse
 maya
 PyYAML
 requests
+retry
```

Comment:
This seems to be incomplete, I think it should somehow go into `requirements_frozen.txt` as well. Fails like this in CI: https://github.com/hiboxsystems/marge-bot/actions/runs/3747611909/jobs/6364021290#step:5:165

Reply:
Good point, added it and an entry to `requirements_frozen.txt` too. :drum:

Reply:
Rebased on top of master and added `retry2` as a Poetry dependency instead.

Reply:
The test is still not working though.. So need to look into that before this can be merged.